src/domain/networking/packets/EntityData.ts

```diff
@@ -9,31 +9,2064 @@
+            animationAllowTranslation = Boolean(data.getUint8(dataPosition));
+            dataPosition += 1;
+        }
+
+        let animationFPS: number | undefined = undefined;
+        if (propertyFlags.getHasProperty(EntityPropertyFlags.PROP_ANIMATION_FPS)) {
+            animationFPS = data.getFloat32(dataPosition, UDT.LITTLE_ENDIAN);
+            dataPosition += 4;
+        }
+
+        let animationFrameIndex: number | undefined = undefined;
+        if (propertyFlags.getHasProperty(EntityPropertyFlags.PROP_ANIMATION_FRAME_INDEX)) {
+            animationFrameIndex = data.getFloat32(dataPosition, UDT.LITTLE_ENDIAN);
+            dataPosition += 4;
+        }
+
+        let animationPlaying: boolean | undefined = undefined;
+        if (propertyFlags.getHasProperty(EntityPropertyFlags.PROP_ANIMATION_PLAYING)) {
+            animationPlaying = Boolean(data.getUint8(dataPosition));
+            dataPosition += 1;
+        }
+
+        let animationLoop: boolean | undefined = undefined;
+        if (propertyFlags.getHasProperty(EntityPropertyFlags.PROP_ANIMATION_LOOP)) {
+            animationLoop = Boolean(data.getUint8(dataPosition));
+            dataPosition += 1;
+        }
+
+        let animationFirstFrame: number | undefined = undefined;
+        if (propertyFlags.getHasProperty(EntityPropertyFlags.PROP_ANIMATION_FIRST_FRAME)) {
+            animationFirstFrame = data.getFloat32(dataPosition, UDT.LITTLE_ENDIAN);
+            dataPosition += 4;
+        }
+
+        let animationLastFrame: number | undefined = undefined;
+        if (propertyFlags.getHasProperty(EntityPropertyFlags.PROP_ANIMATION_LAST_FRAME)) {
+            animationLastFrame = data.getFloat32(dataPosition, UDT.LITTLE_ENDIAN);
+            dataPosition += 4;
+        }
+
+        let animationHold: boolean | undefined = undefined;
+        if (propertyFlags.getHasProperty(EntityPropertyFlags.PROP_ANIMATION_HOLD)) {
+            animationHold = Boolean(data.getUint8(dataPosition));
+            dataPosition += 1;
+        }
+
+        return {
+            bytesRead: dataPosition - position,
+            properties: {
+                shapeType,
+                compoundShapeUrl,
+                color,
+                textures,
+                modelUrl,
+                modelScale,
+                jointRotationsSet,
+                jointRotations,
+                jointTranslationsSet,
+                jointTranslations,
+                groupCulled,
+                relayParentJoints,
+                blendShapeCoefficients,
+                useOriginalPivot,
+                animationUrl,
+                animationAllowTranslation,
+                animationFPS,
+                animationFrameIndex,
+                animationPlaying,
+                animationLoop,
+                animationFirstFrame,
+                animationLastFrame,
+                animationHold
```

Comment:
Animation properties should be grouped in a `animation` property, like the native scripting API is: https://apidocs.vircadia.dev/Entities.html#.EntityProperties-Model